docs/source/rclone.rst

```diff
@@ -0,0 +1,420 @@
+=======================================
+Awscli/Rclone Submission (experimental)
+=======================================
+
+For some use cases we are trialing a more streamlined submission process that does
+not involve the installation of Python or SubmitCGAP and instead involves transferring
+files directly into a `PUBLIC-WRITE` S3 bucket defined by us. The idea is you will source
+credentials associated with the datastore where your files are kept and sync them
+directly into the submission bucket, along with the submission Excel (for data
+accessioning).
+
+These instructions are intended to support both `awscli` if transferring directly from
+`AWS S3` or `rclone` if you are transferring from other Cloud Providers (still compatible
+with `AWS S3`). At this time we recommend using `rclone`, since it is compatible with
+most popular Cloud Providers.
+
+
+Installing rclone
+-----------------
+
+Navigate to the `rclone` `download page <https://rclone.org/downloads/>`_ and click the
+download button that corresponds to the operating system you are running. Once installed
+you should see the `rclone` command in your path.
+
+.. code-block:: bash
+
+    $ which rclone
+
+
+Configuring Anonymous S3 Remote
+-------------------------------
+
+The `rclone` `S3 documentation page <https://rclone.org/s3/>`_ has detailed information
```

Comment:
Unless .rst files have relaxed the restriction, I believe double-backquote is what you're meant to use to get code font. PyCharm when in source editing mode highlights single backquotes in green as if they're some other kind of thing (which if they have adequate hyperlink information they are). Please push back if you think that's wrong.

```suggestion
The ``rclone``
`S3 documentation page <https://rclone.org/s3/>`_
has detailed information
```

I see there are more such things farther up. Maybe we can review those, too, without my having to mark all of them. It's possible that something notices non-hyperlink situations and just tolerates them as meaning code font, but that seems dicey to me since there's a weird ambiguity thing where single backquote with hyperlink markers is not code font.